remoteappmanager/static/js/gamodule.js

```diff
@@ -0,0 +1,26 @@
+// This module contains the setup for google analytics.
+// MUST not be renamed to analytics. Some blockers rely on name 
+// matching to prevent loading.
+define([
+], function () {
+    "use strict";
+
+    function init() {
+        var module;
+        
+        if (window.apidata.analytics !== undefined) {
+            window.ga('create', window.apidata.analytics.tracking_id, 'auto');
+        } else {
+            window.ga = function () {
```

Comment:
pycharm complains for the exact opposite reason. It claims return is unnecessary.
by the way, I use jshint, not jslint (jslint is the one by douglas crockford, and while I respect the guy, I think its linter is a bit excessive)